### PR TITLE
Changes to when and how the player row cards are shown

### PR DESCRIPTION
**Pull request by jozefKruszynski**

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -254,11 +255,22 @@ onMounted(async () => {
   });
 });
 
+const playerStateOrder = {
+  [PlayerState.PLAYING]: 1,
+  [PlayerState.PAUSED]: 2,
+  [PlayerState.IDLE]: 3,
+} as const;
+
```
Comment (marcelveldt) [inline src/views/HomeView.vue]: I thought we agreed on treating paused and idle the same ?

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -268,12 +280,22 @@ const updateCurrentlyPlayingQueueWidgetRow = function () {
           )
         ) {
           widgetRows.value.queue.players.push(player);
+          widgetRows.value.queue.players.sort((a, b) =>
+            a.name.localeCompare(b.name),
```
Comment (marcelveldt) [inline src/views/HomeView.vue]:
```suggestion
            a.display_name.localeCompare(b.display_name),
```

Comment (marcelveldt) [reply src/views/HomeView.vue]: always use display_name as that takes into account any custom name the user gave to a player

Comment (jozefKruszynski) [reply src/views/HomeView.vue]: Will do

**Review by marcelveldt — APPROVED** [review]
Nice!

**Pull request merged**